### ISSUE TITLE
Username of the person who checked in the code

### ISSUE THREAD

**Issue by maheshjwagh** [amoerie/teamcity-theatre#9]
Would you be able to add the Username of the person who checked in the code and triggered the build?

**Comment by amoerie** [comment]
That's definitely possible, but I'm very careful to add anything on the screen, as it needs to be readable from quite a distance. 

A possible solution might be to show the username only when someone breaks a build, but even then it would have to be with some kind of overlay, because there is no remaining free space inside a tile.

**Comment by KrupaMoliya** [comment]
I would like to add username of last changes in can of failed build
How can I get last changes username?

I tried with below way which did't workout.
Added one function  vm.getLastBuild = function (build) {} in View,js 
Call it from View.html like vm.getLastBuild(build);
but in view.js file, build.LastChanges property is an array with 0 length so i am not able to get username.

I think some changes required in GetBuildsOfBuildConfigurationAsync() in "TeamCityTheatre.Application\DataServices\BuildDataService.cs".
